.github/DISCUSSION_TEMPLATE/q-a.yml

```diff
@@ -51,3 +51,4 @@ body:
         - Visual Studio for Mac

```

Comment:
@jeromelaban Visual Studio for Mac is mentioned here but not on [.github/ISSUE_TEMPLATE/bug-report.yml](https://github.com/unoplatform/uno/pull/17437/commits/ef73db11a953ca15f7c874f43d6900b0235658e1#diff-7c49a365610d935ce439e3f241cd3f5d1df89a95d1a674d11ebdb1e518c7335c).
Should be aligned right?